src/main/java/org/walkmod/WalkModFacade.java

```diff
@@ -219,6 +223,58 @@ public WalkModFacade() {
 		this(false, true, false);
 	}
 
+
+	/**
+	 * Initalizes a Walkmod service
+	 *
+	 * @param walkmodCfg
+	 *            Walkmod configuration file. If null, file named 'walkmod.xml'
+	 *            is searched at the root.
+	 * @param options
+	 *            Map of option. See {@link Options} and {@link OptionsBuilder}
+	 *            for available options and default values.
+	 * @param configurationProvider
+	 *            Configuration provider responsible for the resolution of
+	 *            plugins (used to use custom classloading strategies).
+	 *            If null Ivy is used.
+	 */
+	public WalkModFacade(File walkmodCfg, Map<String,Object> options, ConfigurationProvider configurationProvider) {

```

Comment:
We can have both methods. I like the Map option to offer some syntantic sugar, for instance, in groovy you coud do something like this:
```
def facade = new WalkModFacade ("some/path" as File, {offline:true, include: ['one', 'two']}, null)
```

wdyt?